src/components/order-service/order-service-plans.html

```diff
@@ -0,0 +1,22 @@
+<!--
+  TODO:
+  - Improve plan display
+  - Skip plan selection if only one
+-->
+<div ng-repeat="plan in $ctrl.serviceClass.resource.plans" class="col-md-6">
+  <div class="title">Plan {{plan.name}}</div>
+  <!-- TODO : truncate long text -->
+  <div ng-if="plan.description" class="sub-title first">{{plan.description}}</div>
```

Comment:
I'm thinking we may want to consider having plan.osbMetadata.bullets show up underneath the plan description, bullets should be a ul of some kind.  a TODO here would be good for now.